docs/learn/learn-bridges.md

```diff
@@ -106,22 +106,22 @@ Please read the blog article for fuller descriptions of each one of these option
 
 ### Bitcoin Bridge (XCLAIM <-> Substrate <-> Polkadot)
 
-The Interlay team has written a [specification](https://interlay.gitlab.io/polkabtc-spec/) on a
+The Interlay team has written a [specification](https://interlay.gitlab.io/iBTC-spec/) on a
```

Comment:
The provided link requests for a sign-in. Linking to this instead - https://spec.interlay.io/ 

```suggestion
The Interlay team has written a [specification](https://spec.interlay.io/) on a
```